py-notebook/jupyter-widget/js/lib/NeoWidget.js

```diff
@@ -0,0 +1,78 @@
+var widgets = require("@jupyter-widgets/base");
+var _ = require("lodash");
```

Comment:
Where does this get used?

Reply:
Nowhere so far, but it may be related to the TODO tasks still pending?